jupyterlab_git/git.py

```diff
@@ -249,20 +249,24 @@ def __init__(self, contents_manager):
             }
 
         result = []
-        line_array = my_output.splitlines()
-        for line in line_array:
-            to1 = None
-            from_path = line[3:]
-            if line[0] == "R":
-                to0 = line[3:].split(" -> ")
-                to1 = to0[len(to0) - 1]
+        line_iterable = iter(my_output.strip("\x00").split('\x00'))
+        result = []
+        for line in line_iterable:
+            x = line[0]
```

Comment:
why move these into a var if they are only used in the one location? I would leave them inside the append function as they were before